src/mgmt/pool.pb-c.h

```diff
@@ -10,42 +10,42 @@ PROTOBUF_C__BEGIN_DECLS
 
 #if PROTOBUF_C_VERSION_NUMBER < 1003000
 # error This file was generated by a newer version of protoc-c which is incompatible with your libprotobuf-c headers. Please update your headers.
-#elif 1003000 < PROTOBUF_C_MIN_COMPILER_VERSION
```

Comment:
I'm not sure about all of the changes in this auto-generated file, whether they are all needed. @kjacque would you be able to comment on that a little better than I can?

Reply:
The protobuf changes all look okay to me. There can be a bit of flux depending on the exact version of protoc and/or the protoc-c plugin was used, but it doesn't really matter as long as it doesn't impact the callers.

Reply:
Yes, i found it harder to really port "auto-generated" files, and it might be wrong for porting, so instead i just ported proto file and then regenerated these files.

Let me know if you have better ideas how we could handle this.

Reply:
@wangshilong Regenerating the files is the right approach. No better way to handle it that I'm aware of.